server/app/Icepeak.hs

```diff
@@ -51,7 +52,8 @@ readValue filePath = do
 
 main :: IO ()
 main = do
-  config <- execParser configInfo
+  env <- getEnvironment
+  config <- execParser (configInfo env)
```

Comment:
When starting up, we should log whether authorization is enabled, and whether a secret has been given (but not the actual key). This because not enabling authorization is a possible security problem.

Reply:
Done!